book/website/reproducible-research/licensing/licensing-ml.md

```diff
@@ -2,17 +2,17 @@
 # Machine Learning Model Licenses
 
 Like a software license, a Machine Learning (ML) model license governs the use, redistribution of the model and/or algorithm, and distribution of any derivatives of it. However, there are other components to an AI system, such as [data](https://the-turing-way.netlify.app/reproducible-research/licensing/licensing-data.html), 
-[source code](https://the-turing-way.netlify.app/reproducible-research/licensing/licensing-software.html), or applications, which may have their own separate licenses. ML model licenses may restrict the use of the model for specific scenarios for which, due to ethics-informed concerns and/or technical limitations of the model informed by its model card, the licensor is not comfortable that the model is used.
+[source code](https://the-turing-way.netlify.app/reproducible-research/licensing/licensing-software.html), or applications, which may have their own separate licenses. ML model licenses may restrict the use of the model for specific scenarios for which, due to the technical capabilities and limitations  of the model informed by its model card, the licensor is not comfortable that the model is used.
```

Comment:
```suggestion
[source code](https://the-turing-way.netlify.app/reproducible-research/licensing/licensing-software.html), or applications, which may have their own separate licenses. ML model licenses may restrict the use of the model for specific scenarios for which, due to the technical capabilities and limitations of the model informed by its model card, the licensor is not comfortable that the model is used.
```
I'm not sure how I missed this before (sorry!), but it'd be great to have a short description here of 'model card'. Maybe something like "technical capabilities and limitations of the model informed by its model card – a term used to describe summaries of ML models" perhaps?